core/scss/components/site-search/_site-search.scss

```diff
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+//
+// Site Search
+//
+// A global site search component
+//
+// Markup: ../templates/components/site-search/site-search.twig
+//
+// Style guide: Components.Site Search
+//
+.su-site-search__input {
+  @include font-smoothing();
+  @include padding(.3em 1em);
+
+  display: inline-block;
+  height: 40px;
```

Comment:
Would min-height be better here?
Would the value be better specified in (r)em's?

Reply:
I'm leaving the height as px for now, since in Figma the height of the bar stays the same for all break points, and using em might change the height for different break points. I'm not super up-to-date with the typography settings - is rem always going to be 10px?